upf/upf_pfcp_api.c

```diff
@@ -2570,6 +2608,9 @@ handle_session_establishment_request (pfcp_msg_t * msg,
       pending->inactivity_timer.handle = ~0;
     }
 
+  if (ISSET_BIT (req->grp.fields, SESSION_ESTABLISHMENT_REQUEST_USER_ID))
+    memcpy (&sess->user_id, &req->user_id, sizeof (pfcp_user_id_t));
```

Comment:
That doesn't work for  the `nai` field. We don't use it at the moment, but to avoid problems in the future it would be better initialize it to zero in the copy.
If you wanted to copy it as well, then it needs to freed together with the session